src/backend/mpi/swdsm.cpp

```diff
@@ -78,7 +78,7 @@ int numtasks;
 /** @brief  rank/process ID in the MPI/ArgoDSM runtime*/
 int rank;
 /** @brief rank/process ID in the MPI/ArgoDSM runtime*/
-int workrank;
+argo::node_id_t workrank;
```

Comment:
It seems strange to change the type of `workrank` and leave `rank` declared `int`.  Aren't these two related?

Reply:
The only use of `rank` in the backend is to run `MPI_Comm_rank`. The whole way we initialize MPI should probably be re-written as I discovered there is more "dead code" related to this, and `rank` is one of the things that is never used in the code outside of the mentioned call.